src/elements/pages/dev/game-banner.ts

```diff
@@ -0,0 +1,36 @@
+import { LitElement, TemplateResult, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import styles from './game-banner.scss';
+import * as api from '../../../utils/api';
+import * as cloud from '@rivet-gg/cloud';
+import { cssify } from '../../../utils/css';
+import assets from '../../../data/assets';
+
+@customElement('game-banner')
+export default class DevGameBanner extends LitElement {
+	static styles = cssify(styles);
+
+	@property({ type: Object })
+	game: cloud.GameFull;
+
+	renderGameIcon(game: cloud.GameFull): TemplateResult {
+		return html`
+			<lazy-img
+				class="mx-auto w-28 h-28"
+				bg-size=${game.logoUrl ? 'contain' : 'cover'}
+				src=${game.logoUrl ?? assets.asset('/games/blank/logo.png')}
+			></lazy-img>
+		`;
+	}
```

Comment:
Nit: This doesn't need to be a separate render function unless it gets reused, probably better left inline.